public/app/plugins/panel/xychart/v2/utils.ts

```diff
@@ -107,8 +107,12 @@ export function prepSeries(
       if (x != null) {
         // match y fields and create series
         onlyNumFields.forEach((field) => {
-          // don't reuse already-mapped fields
-          if (field === x || field === color || field === size) {
```

Comment:
found a real dashboard where y value was also used for color field (with 11 thresholds :exploding_head:). sp relaxed this to be possible in Manual mode.